labs/create-producer-node/server.js

```diff
@@ -1,7 +1,7 @@
 const mqtt = require('mqtt');
 
-//<prefix><version>/journey/<temporal_type>/<transport_mode>/<operator_id>/<vehicle_number>/<route_id>/<direction_id>/<headsign>/<start_time>/<next_stop>/<geohash_level>/<geohash>/
-const topic = '/hfp/v1/journey/ongoing/+/+/+/+/+/+/+/+/+/#';
+//<prefix>/<version>/<journey_type>/<temporal_type>/<event_type>/<transport_mode>/<operator_id>/<vehicle_number>/<route_id>/<direction_id>/<headsign>/<start_time>/<next_stop>/<geohash_level>/<geohash>/#
+const topic = '/hfp/v2/journey/ongoing/vp/+/+/+/+/+/+/+/+/+/#';
```

Comment:
We *could* reduce the bandwidth we are using a geohash_level in this topic.  If we change this we would also need to push an update to the `vp-producer` container.

Reply:
Do we know if this would increase the chance of getting no data back at certain times of day? Just wondering if we want to explore this change now, or defer until later.